### PR TITLE
Add link to website of TiKV's creator

### LINKED ISSUE #12702: Add website of creator of TiKV to README.md
Add website of creator of TiKV to README.md

### PR DESCRIPTION
**Pull request by sunxiaoguang**
Signed-off-by: Xiaoguang Sun <sunxiaoguang@gmail.com>

### What is changed and how it works?
Add link to website of TiKV's creator

Issue Number: Close #12702 

What's Changed:
Add link to website of TiKV's creator in README.md

```Add link to website of TiKV's creator
```

### Check List 

- No code

Side effects

### Release note 

```Add link to website of TiKV's creator
```

### CODE REVIEW

**Review by dcalvin — APPROVED** [review]
LGTM

**Review by winkyao — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [![Coverage Status](https://codecov.io/gh/tikv/tikv/branch/master/graph/badge.svg)](https://codecov.io/gh/tikv/tikv)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/2574/badge)](https://bestpractices.coreinfrastructure.org/projects/2574)
 
-TiKV is an open-source, distributed, and transactional key-value database. Unlike other traditional NoSQL systems, TiKV not only provides classical key-value APIs, but also transactional APIs with ACID compliance. Built in Rust and powered by Raft, TiKV was originally created to complement [TiDB](https://github.com/pingcap/tidb), a distributed HTAP database compatible with the MySQL protocol.
+TiKV is an open-source, distributed, and transactional key-value database. Unlike other traditional NoSQL systems, TiKV not only provides classical key-value APIs, but also transactional APIs with ACID compliance. Built in Rust and powered by Raft, TiKV was originally created by [PingCAP](https://en.pingcap.com) created to complement [TiDB](https://github.com/pingcap/tidb), a distributed HTAP database compatible with the MySQL protocol.
```
Comment (BusyJay) [inline README.md:9]: The sentence seems grammatically wrong to me.

Comment (dcalvin) [inline README.md:9]:
```suggestion
TiKV is an open-source, distributed, and transactional key-value database. Unlike other traditional NoSQL systems, TiKV not only provides classical key-value APIs, but also transactional APIs with ACID compliance. Built in Rust and powered by Raft, TiKV was originally created by [PingCAP](https://en.pingcap.com) to complement [TiDB](https://github.com/pingcap/tidb), a distributed HTAP database compatible with the MySQL protocol.
```

Comment (sunxiaoguang) [reply README.md]: Good catch, fixed thanks.

**Review by ngaut — APPROVED** [review]

**Review by BusyJay — APPROVED** [review]

**Pull request merged**